atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -167,6 +168,11 @@ def make_labels(self):
         self._update_labels(labels, self.workflow.labels)
         self._update_labels(labels, self.workflow.build_result.labels)
 
+        if ('sources_for_nvr' in labels and len(labels['sources_for_nvr']) > LABEL_MAX_CHARS):
```

Comment:
Could we instead drop this label altogether?

My concern is that if we decide to use this label for something meaningful in the future, we'll most likely have forgotten that in some cases it could be an empty value.

Reply:
As an alternative, we can use `sources_for_build_id` instead.

Reply:
changed label to sources_for_koji_build_id